deploy/kubernetes/examples/demo-pvc-file-system.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 metadata:
   name: demo-pvc
 spec:
+  volumeMode: FileSystem
```

Comment:
change to [`Filesystem`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes) IN ALL YAMLS